lib/players/human.rb

```diff
@@ -0,0 +1,13 @@
+
+class Player::Human < Player 
+ 
+def move(arr)
+  
+input = gets.strip 
+
+end 
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  gets.strip
end
```